### ISSUE TITLE
I can't get the reconstruction results to be comparably good to what is shown in the original paper

### ISSUE THREAD

**Issue by josef5838** [maurock/DeepSDF#5]
Hi @maurock thanks for your amazing work! It has helped me a lot with my current project.

However, when I train and run your deepsdf implementation on the bus subset of shapenet, my results are unfortunately bad.

I have used the same setup as yours in your project touchsdf:
`# training
epochs: 1000
lr_model: 0.00001   # Initial learning rate (model)
lr_latent: 0.001    # Initial learning rate (latent vector)
batch_size: 20480
sigma_regulariser: 0.01   # Sigma value as regulariser in the loss function
loss_multiplier: 1   # Loss multiplier
clamp: true  # Clip the network prediction
clamp_value: 0.1  # Value for clipping

# architecture
num_layers: 8   # Num network layers
inner_dim: 512   # Inner dimensions of the network
latent_size: 128   # Size of the latent vector
skip_connections: True`

The original bus mesh in the dataset appears as follows:
<img width="445" alt="image" src="https://github.com/maurock/DeepSDF/assets/92803422/de1387f0-9629-4d67-a7c2-c5335d21e447">

However, the results after applying the trained weights and extracting meshes are not ideal:
<img width="598" alt="image" src="https://github.com/maurock/DeepSDF/assets/92803422/b5cdb42c-baa3-4cee-8758-53ecb6ea36fb"

I would greatly appreciate any insights or suggestions you might have for improving these outcomes. Additionally, if possible, could you share the weights trained on the full ShapeNet dataset?

Your help is sincerely appreciated,
best,
Haoliang
